package.json

```diff
@@ -8,10 +8,12 @@
     "precommit": "lint-staged",
     "commit": "git-cz",
     "test": "jest --coverage",
+    "posttest": "npm run test-examples",
     "lint": "standard --verbose | snazzy",
     "contributors:add": "all-contributors add",
     "publish": "yarn run lint && jest --coverage && lerna publish --conventional-commits",
-    "watch": "jest --watchAll --coverage"
+    "watch": "jest --watchAll --coverage",
+    "test-examples": "node other/run-examples.js"
```

Comment:
on the file you have it marked as an executable, change the file `chmod +x` and you can execute it as is without running it through node

Reply:
can u also change the name of the npm script to `run-examples` to  match the name of the file